code/modules/mob/living/living_defense.dm

```diff
@@ -194,6 +194,8 @@
 /mob/living/proc/adjust_fire_stacks(add_fire_stacks) //Adjusting the amount of fire_stacks we have on person
 	if(status_flags & GODMODE) //Invulnerable mobs don't get fire stacks
 		return
+	if(add_fire_stacks)	//Fire stack increases are affected by armor, end result rounded up.
+		add_fire_stacks = CEILING(fire_stacks * get_fire_resist(), 1)
```

Comment:
did you want this to be ceiling(add_fire_stacks ?

Reply:
oops